.trunk/trunk.yaml

```diff
@@ -42,14 +42,23 @@ actions:
         - git_hooks: [pre-push]
 
     - id: linter-test-helper
-      display_name: Test Generator
+      display_name: Linter Test Generator
       description: Generate boilerplate test code when new linters are defined
```

Comment:
Do we want the linter boilerplate to now include tools?

Reply:
done, good idea